### PR TITLE
Change to biodata catalyst branding [SATURN-1331]

### PR DESCRIPTION
**Pull request by cameron-ardell**
DataSTAGE’s name has changed to BioData Catalyst.  NHLBI Leadership now requires us to change name and logo across the various website pages that currently exist within Terra.

This is almost there, but I haven't figured out yet how to wrap the text for the longer name (and we don't have a shorter version) and selectively make just the logo for this larger.

UPDATE 1/30 2:15p - Logos updated. ready for full review.

### CODE REVIEW

File: src/libs/logos.js [hunk 1]
```diff
@@ -1,30 +1,35 @@
...
-import { isAnvil, isDatastage, isFirecloud, isTerra } from 'src/libs/config'
+import { isAnvil, isBioDataCatalyst, isDatastage, isFirecloud, isTerra } from 'src/libs/config'
 import * as Utils from 'src/libs/utils'
 
 
 export const getAppName = (longName = false) => Utils.cond(
   [isFirecloud(), 'FireCloud'],
   [isDatastage(), 'DataStage'],
   [isAnvil(), longName ? 'The NHGRI AnVIL (Genomic Data Science Analysis, Visualization, and Informatics Lab-space)' : 'AnVIL'],
+  [isBioDataCatalyst(), 'NHLBI BioData CATALYST'],
```
Comment (breilly2) [inline src/libs/logos.js]: Should be `'NHLBI BioData Catalyst'`.
Unlike the logo, when the words are written out, "Catalyst" is not all uppercase. (https://www.nhlbidatastage.org/)

File: src/images/brands/bioDataCatalyst/bioDataCatalyst-Logo-dark.svg [hunk 2]
```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (breilly2) [inline src/images/brands/bioDataCatalyst/bioDataCatalyst-Logo-dark.svg]: It looks like we're not using this logo variant anywhere.

File: src/libs/logos.js [hunk 3]
```diff
@@ -1,30 +1,35 @@
...
   'Terra'
 )
 
 export const returnParam = () => getAppName().toLowerCase()
 
+// do io add here is Terra
```
Comment (breilly2) [inline src/libs/logos.js]: Accidental commit?

**Review by breilly2 — COMMENTED** [review]
A couple of comments, but overall looks good. Will hold approval for getting better SVG assets in place.

File: src/libs/logos.js (hunk 1 above)
Comment (cameron-ardell) [reply src/libs/logos.js]: It's in all capital letters in the inVision docs (can send you a screenshot if you don't have permissions)

File: src/libs/logos.js (hunk 3 above)
Comment (cameron-ardell) [reply src/libs/logos.js]: d'oh. yes.

File: src/libs/logos.js (hunk 1 above)
Comment (breilly2) [reply src/libs/logos.js]: I checked with Lou who checked their style guide and found that it shouldn't be YELLING. He updated the mocks.

**Review by ehanna4 — APPROVED** [review]
Looks good, tested by using config overrides to view in browser and looked over code. Did not test if the url directs to the correct site

**Pull request merged**